containers/ddev-webserver/files/usr/local/bin/wp

```diff
@@ -2,6 +2,12 @@
 ##
 # WP-cli wrapper: Append path automatically so that user doesn't have to
 ##
+
+PATH_OPTION=""
+if [ -z "$WP_CORE" ]; then
+  PATH_OPTION="--path=$WP_CORE"
```

Comment:
This actually doesn't make sense - it says "If WP_CORE is not set, then use --path=$WP_CORE". So it's not right as I read it. Did you mean `if [  ! -z "$WP_CORE" ]` ?

Reply:
Yes, you are right!

I changed it to `[ -n "$WP_CORE" ]`.